lib/endpoints/class-wp-json-attachments-controller.php

```diff
@@ -144,33 +144,45 @@ protected function prepare_item_for_database( $request ) {
 	 */
 	public function prepare_item_for_response( $post, $request ) {
 		$response = parent::prepare_item_for_response( $post, $request );
+		$data = $response->get_data();
 
-		$response['alt_text']      = get_post_meta( $post->ID, '_wp_attachment_image_alt', true );
-		$response['caption']       = $post->post_excerpt;
-		$response['description']   = $post->post_content;
-		$response['media_type']    = wp_attachment_is_image( $post->ID ) ? 'image' : 'file';
-		$response['media_details'] = wp_get_attachment_metadata( $post->ID );
-		$response['post']          = ! empty( $post->post_parent ) ? (int) $post->post_parent : null;
-		$response['source_url']    = wp_get_attachment_url( $post->ID );
+		$data['alt_text']      = get_post_meta( $post->ID, '_wp_attachment_image_alt', true );
+		$data['caption']       = $post->post_excerpt;
+		$data['description']   = $post->post_content;
+		$data['media_type']    = wp_attachment_is_image( $post->ID ) ? 'image' : 'file';
+		$data['media_details'] = wp_get_attachment_metadata( $post->ID );
+		$data['post']          = ! empty( $post->post_parent ) ? (int) $post->post_parent : null;
+		$data['source_url']    = wp_get_attachment_url( $post->ID );
```

Comment:
Because the parent post controller `prepare_item_for_response` now returns an object instead of an array.  I renamed the variable because we are modifying the response data not the response itself.